PROXY-GUIDE.md

```diff
@@ -50,11 +50,30 @@ For that, you can try forcing the connection to bypass the proxy by setting the
 NxAPI pods.
 
 
+## Self-Signed SSL Certificates
 
-## Supporting Self-Signed SSL Certificates
+If you deploy the `nx-cloud-frontend` pod with a self-signed certificate you can use the same strategy as in the case of running behind a proxy, but this time you can use `node:https`. Change the contents of the `custom-proxy-config.js` file detailed above to the following:
```

Comment:
```suggestion
If you have set-up your NxCloud cluster a self-signed certificate you can use the same strategy as in the case of running behind a proxy, but this time you can use `node:https`. Change the contents of the `custom-proxy-config.js` file detailed above to the following:
```

Reply:
we can generalise this, just because it wouldn't really be the `frontend` pod, but the runner would connect to the `nx-api` pod via the cluster's LB - and it depends where TLS is terminated